static/js/click_handlers.js

```diff
@@ -249,6 +244,12 @@ exports.initialize = function () {
         e.stopPropagation();
         e.preventDefault();
     });
+    $('body').on('click', '.always_visible_topic_edit,.on_hover_topic_edit', function (e) {
```

Comment:
Shouldn't this be using the `message_edit_notice` class for the click handler here?

Reply:
have used .message_edit_notice class only